src/main/java/org/apache/commons/text/names/HumanNameParser.java

```diff
@@ -98,8 +98,9 @@
  *
  * <p>This class is immutable.</p>
  */
-public final class HumanNameParser {
+public class HumanNameParser {
```

Comment:
That'd be great Tom. Other than that, I have no other observations about the pull request, and believe we are ready to merge it once you have reverted the changes :-D